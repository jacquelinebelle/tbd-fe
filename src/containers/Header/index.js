import React from 'react';
import { Link } from 'react-router-dom';
import './Header.scss'

const Header = () => {
  return (
    <header> 
        <Link to='/' className='logo'>
          <img alt="Compass site logo with title COMPASS"src={require('./compassLogo-test1.png')} className='logo-img'/>
        </Link>
    </header>
  )
}

export default Header;