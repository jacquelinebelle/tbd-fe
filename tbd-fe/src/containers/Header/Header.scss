header {
  align-items: center;
  background-color: #faf9f5;
  display: flex;
  justify-content: center;
  margin-bottom: 50px;
  padding: 10px 50px;
}

.logo {
  align-items: center;
  display: flex;
  justify-content: center;
  width: 60%;
}

.logo-img {
  width: 40%;
  min-height: 32px;
  min-width: 169px;
}