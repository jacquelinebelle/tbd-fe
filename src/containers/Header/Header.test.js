import React from "react";
import { shallow } from "enzyme";
import Header from "./index";

describe("Header", () => {
  let wrapper;

  beforeEach(function() {
    wrapper = shallow(<Header />);
  });

  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
