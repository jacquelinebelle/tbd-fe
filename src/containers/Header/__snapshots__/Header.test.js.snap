// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<header>
  <Link
    className="logo"
    to="/"
  >
    <img
      className="logo-img"
      src="compassLogo-test1.png"
    />
  </Link>
</header>
`;
